ludwig/config_validation/checks.py

```diff
@@ -606,3 +607,38 @@ def check_qlora_requirements(config: "ModelConfig") -> None:  # noqa: F821
 
     if config.quantization and (not config.adapter or config.adapter.type != "lora"):
         raise ConfigValidationError("Fine-tuning and LLM with quantization requires using the 'lora' adapter")
+
+
+@register_config_check
+def check_llm_template_references(config: "ModelConfig") -> None:  # noqa: F821
+    """Checks that prompt.template includes a valid reference to a feature column."""
+    if config.model_type != MODEL_LLM:
+        return
+
+    template = config.prompt.template
```

Comment:
I'd add a check to make sure `config.prompt` is not None first, and then proceed with this check

Reply:
out of date